ddl/generated_column.go

```diff
@@ -198,6 +207,34 @@ func checkIllegalFn4GeneratedColumn(colName string, expr ast.ExprNode) error {
 	return nil
 }
 
+// Check whether newColumnDef refers to any auto-increment columns.
+func checkGeneratedWithAutoInc(tableInfo *model.TableInfo, newColumnDef *ast.ColumnDef) error {
+	_, dependColNames := findDependedColumnNames(newColumnDef)
+	if err := checkAutoIncrementRef(newColumnDef.Name.Name.L, dependColNames, tableInfo); err != nil {
+		return errors.Trace(err)
+	}
+	return nil
+}
+
+func checkIndexOrStored(tbl table.Table, oldCol, newCol *table.Column) error {
```

Comment:
One function do one thing, please check stored in another function.

Reply:
@winkyao I agreed with you. But once backfilling is supported, we can remove the whole function entirely. Additionly, it is short and a part of logic can be reused. How do you think?